config/config.go

```diff
@@ -11,10 +11,17 @@ import (
 var ErrInvalidTransport = fmt.Errorf("invalid transport")
 
 type GlobalConfig struct {
-	CachePath string `yaml:"cache_path"`
+	CachePath           string `yaml:"cache_path"`
+	DisableLoopDevice   bool   `yaml:"no_loop_device"`
+	BuildkitCacheSizeMB int    `yaml:"buildkit_cache_size_mb"`
```

Comment:
Did you mean to remove `buildkit_` prefix here?

Reply:
I'm not sure to be honest. Should we keep it as `buildkit_cache_size_mb` or rename it to `cache_size_mb` (which makes it more in line with the `cache_path` config).

Reply:
in the end I removed the prefix so it (`cache_size_mb`) aligns with `cache_path`.